src/app/common/components/formfield-image.ts

```diff
@@ -473,8 +473,9 @@ export function issuerImageLoader(file: File | string): Promise<string> {
 		return readFileAsDataURL(file)
 			.then(loadImageURL)
 			.then((image) => {
-
-				if(image.width / image.height !== 1) {
+				
+				const tolerance = 0.05;
+				if((Math.abs(image.width / image.height - 1) > tolerance)) {
 					return Promise.reject(new Error('Image must be square'));
```

Comment:
Should the error message then also indicate that it doesn't have to be *exactly* square?

Reply:
The error message was given to me by mycelia, i would prefer to leave it as is for now